internal/kafka/internal/workers/cluster_mgrs/dynamic_scaleup_mgr.go

```diff
@@ -70,77 +69,14 @@ func (m *DynamicScaleUpManager) Reconcile() []error {
 	glog.Infoln("running dynamic scale up reconcile event")
 	defer m.logReconcileEventEnd()
 
-	// TODO remove this method call and the method itself the new dynamic scale up
-	// logic is ready
-	err := m.reconcileClustersForRegions()
+	err := m.processDynamicScaleUpReconcileEvent()
 	if err != nil {
-		errList.AddErrors(err...)
-	}
-
-	// TODO remove the "if false" condition once the new dynamic scale up
-	// logic is ready
-	if false {
-		err := m.processDynamicScaleUpReconcileEvent()
-		if err != nil {
-			errList.AddErrors(err)
-		}
+		errList.AddErrors(err)
```

Comment:
```suggestion
		return err.ToErrorSlice()
```

Reply:
With this suggestion, this variable can be removed https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/c55d140f8ed60d90fa9bb2785acd112f5b2cd70f/internal/kafka/internal/workers/cluster_mgrs/dynamic_scaleup_mgr.go#L64

Reply:
This cannot be done due to `processDynamicScaleUpReconcileEvent` returns an `error` as opposed to the `ErrorList` data type

Reply:
Thanks. You are right. Sorry, I missed this!